src/v2/Apps/Consign/Routes/MarketingLanding/Components/GetPriceEstimate/EstimateResults.tsx

```diff
@@ -16,7 +16,7 @@ export const EstimateResults: React.FC<FlexProps> = ({ ...rest }) => {
 
   return (
     <Flex {...rest}>
-      {isFetching || artistInsights ? (
+      {isFetching || artistInsights.length ? (
```

Comment:
(non-blocking) Wouldn't this mean the zero state won't render since if there are no results we get an empty array? I see from the gif that it's working, so I'm wondering what I'm missing